src/app/members/actions/createUser.ts

```diff
@@ -20,7 +21,7 @@ export const createUser = async (values: UserBuilder) => {
 
   const invite_link = `${protocol}://${host}?token=${randomInvitedId}`;
 
-  const userPhoneNumber = formatPhoneNumber(values.phoneNumber);
+  const userPhoneNumber = formattedPhoneNumber(values.phoneNumber);
```

Comment:
LOL, you mistook my previous comment. What I meant was to update the variable name to `formattedPhoneNumber`. Function name `formatPhoneNumber` was fine.